src/extras-openmoji/healthcare/E311.svg

```diff
@@ -0,0 +1,28 @@
+<svg id="emoji" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 72 72">
+  <title>E311</title>
+  <g id="grid">
+    <path d="M68,4V68H4V4H68m4-4H0V72H72V0Z" fill="#b3b3b3"/>
+    <path d="M12.923,10.9583H59.0769A1.9231,1.9231,0,0,1,61,12.8814V59.0352a1.9231,1.9231,0,0,1-1.9231,1.9231H12.9231A1.9231,1.9231,0,0,1,11,59.0352V12.8813A1.923,1.923,0,0,1,12.923,10.9583Z" fill="none" stroke="#00a5ff" stroke-miterlimit="10" stroke-width="0.1"/>
+    <rect x="16" y="4" width="40" height="64" rx="2.2537" fill="none" stroke="#00a5ff" stroke-miterlimit="10" stroke-width="0.1"/>
+    <rect x="16" y="4" width="40" height="64" rx="2.2537" transform="translate(72) rotate(90)" fill="none" stroke="#00a5ff" stroke-miterlimit="10" stroke-width="0.1"/>
+    <circle cx="36" cy="36" r="29" fill="none" stroke="#00a5ff" stroke-miterlimit="10" stroke-width="0.1"/>
+  </g>
+  <g id="color">
+    <path d="M32.2408,32.6455c-.161-.6615-.2711-1.294-.4723-1.8959a18.1,18.1,0,0,0-.9249-2.3565,3.2039,3.2039,0,0,0-4.43-1.6284,9.7475,9.7475,0,0,0-2.6462,1.6835,26.67,26.67,0,0,0-5.9735,7.645,21.851,21.851,0,0,0-2.2313,5.5814,14.9153,14.9153,0,0,0-.4158,2.8438c-.0734,2.3022-.0728,4.6076-.0614,6.9115a10.9226,10.9226,0,0,0,.4871,3.4047,3.2346,3.2346,0,0,0,3.14,2.4558,1.6351,1.6351,0,0,1,.29.0645h.1519c.3368-.04.6753-.0694,1.01-.122a15.89,15.89,0,0,0,6.6285-2.8513,15.6321,15.6321,0,0,0,4.808-5.139,9.0951,9.0951,0,0,0,1.2411-4.7473q-.0918-3.8892-.1747-7.7786A23.3713,23.3713,0,0,0,32.2408,32.6455Z" fill="#92d3f5"/>
+    <path d="M37.4951,32.326c.0508-.2238.0889-.38.1215-.5365a11.8359,11.8359,0,0,1,1.2788-3.6313,3.2285,3.2285,0,0,1,2.6531-1.8412,4.3263,4.3263,0,0,1,2.22.5526,14.5931,14.5931,0,0,1,4.2462,3.5118,29.3667,29.3667,0,0,1,5.1242,8.2023,16.2156,16.2156,0,0,1,1.4317,6.6242c-.0117.5285.0393,1.0584.0614,1.5876v4.8619c-.0255.1974-.0414.3965-.0782.5918a23.7765,23.7765,0,0,1-.532,2.6653A2.9842,2.9842,0,0,1,51.7,57.0763a6.52,6.52,0,0,1-3.1888-.1633,17.3332,17.3332,0,0,1-10.1151-7.2761,8.9134,8.9134,0,0,1-1.5508-5.1524c.0494-2.6436.119-5.2869.1755-7.93A23.711,23.711,0,0,1,37.4951,32.326Z" fill="#92d3f5"/>
```

Comment:
why are the lungs blue? lungs are definitely red

Reply:
You are right, that's odd. Changed to red.